### PR TITLE
Review V1 Swagger for completeness (System Agent).

### LINKED ISSUE #2365: Review V1 Swagger for Completeness -- System-Agent
All of the old RAML documents for the API endpoints in Core/Supporting services were converted to Swagger by the QA/Test team. As part of this conversion, we identified the need to review the resulting Swagger for completeness. If the original RAML document did not include comprehensive description of a service's API behavior, then obviously the converted Swagger won't either.

This issue is for review of the system-agent V1 [Swagger doc](https://github.com/edgexfoundry/edgex-go/blob/master/api/openapi/v1/system-agent.yaml).

### PR DESCRIPTION
**Pull request by akramtexas**
Fix #2365

### CODE REVIEW

**Comment by akramtexas** [comment]
Rebased

File: api/openapi/v1/system-agent.yaml [hunk 1]
```diff
@@ -8,21 +8,23 @@ paths:
...
         required: true
         style: simple
         explode: false
         schema:
           type: string
       responses:
         200:
-          description: an array of config
+          description: an array of elements where each element contains the
+          configuration data themselves, along with the service name each for which
```
Comment (cherrycl) [inline api/openapi/v1/system-agent.yaml]: Got YAML validation failure in the description section. The description contents should display like line 11 if the contents contain multiple lines. Would you please check the full file?

File: api/openapi/v1/system-agent.yaml [hunk 2]
```diff
@@ -8,21 +8,23 @@ paths:
...
+            comma-delimited)
         required: true
         style: simple
         explode: false
         schema:
           type: string
       responses:
         200:
-          description: an array of config
+          description: an array of elements where each element contains the configuration
```
Comment (brandonforster) [inline api/openapi/v1/system-agent.yaml]: [Let's simplify and make this more concise according to the Swagger example.](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#paths-object)

Something like `A list of configuration data corresponding to each requested, valid service.`

File: api/openapi/v1/system-agent.yaml [hunk 3]
```diff
@@ -32,27 +34,56 @@ paths:
   /v1/metrics/{services}:
     get:
       description: Fetch the operating performance metrics of the specified EdgeX
-        services by their unique names - returning null if none is found for the service.
-        HTTP 500 for unknown or unanticipated issues
+        unique names - returning an error message if a problem is encountered for a
+        given service. And returning HTTP 500 for unknown or unanticipated issues
```
Comment (brandonforster) [inline api/openapi/v1/system-agent.yaml]: Please remove specifics on errors as described on the `config/{services}` endpoint.

File: api/openapi/v1/system-agent.yaml [hunk 4]
```diff
@@ -32,27 +34,56 @@ paths:
...
             '*/*':
               schema:
                 $ref: '#/components/schemas/metric'
         500:
           description: for unknown or unanticipated issues.
+  /v1/health/{services}:
+    get:
+      description: Fetch the health status of the specified EdgeX services by their
+        unique names - returning either true (for a healthy service) or a given
+        service is not healthy. And returning HTTP 500 for unknown or unanticipated
```
Comment (brandonforster) [inline api/openapi/v1/system-agent.yaml]: Please remove specifics on errors as described on the `config/{services}` endpoint.

File: api/openapi/v1/system-agent.yaml [hunk 5]
```diff
@@ -32,27 +34,56 @@ paths:
...
-        services by their unique names - returning null if none is found for the service.
-        HTTP 500 for unknown or unanticipated issues
+        unique names - returning an error message if a problem is encountered for a
+        given service. And returning HTTP 500 for unknown or unanticipated issues
       parameters:
       - name: services
         in: path
-        description: EdgeX micro service names (the unique name of each service comma
-          delimited)
+        description: EdgeX micro service names (the unique name of each service,
```
Comment (brandonforster) [inline api/openapi/v1/system-agent.yaml]: Please avoid the use of parenthesis as described on the `config/{services}` endpoint.

File: api/openapi/v1/system-agent.yaml [hunk 6]
```diff
@@ -32,27 +34,56 @@ paths:
...
+            comma-delimited)
+          required: true
+          style: simple
+          explode: false
+          schema:
+            type: string
+      responses:
+        200:
+          description: either true (for a healthy service) or that a
+            given service is not healthy
```
Comment (brandonforster) [inline api/openapi/v1/system-agent.yaml]: 200 means that it is not healthy?

File: api/openapi/v1/system-agent.yaml [hunk 7]
```diff
@@ -32,27 +34,56 @@ paths:
...
+    get:
+      description: Fetch the health status of the specified EdgeX services by their
+        unique names - returning either true (for a healthy service) or a given
+        service is not healthy. And returning HTTP 500 for unknown or unanticipated
+        issues
+      parameters:
+        - name: services
+          in: path
+          description: EdgeX micro service names (the unique name of each service,
+            comma-delimited)
```
Comment (brandonforster) [inline api/openapi/v1/system-agent.yaml]: Please avoid the use of parenthesis as described on the `config/{services}` endpoint.

File: api/openapi/v1/system-agent.yaml [hunk 8]
```diff
@@ -32,27 +34,56 @@ paths:
...
+      responses:
+        200:
+          description: either true (for a healthy service) or that a
+            given service is not healthy
+          content:
+            '*/*':
+              schema:
+                $ref: '#/components/schemas/health'
+        500:
+          description: for unknown or unanticipated issues.
```
Comment (brandonforster) [inline api/openapi/v1/system-agent.yaml]: Please capitalize the first word of all descriptions.

File: api/openapi/v1/system-agent.yaml [hunk 9]
```diff
@@ -64,6 +95,15 @@ paths:
               $ref: '#/components/schemas/operation'
         required: true
       responses:
+        200:
+          description: an array of elements where each element contains
```
Comment (brandonforster) [inline api/openapi/v1/system-agent.yaml]: This is too much detail. Maybe something like `A list of services and their executors.`? We don't need to mention that these executors were requested; that's implied.

**Pull request merged**